### PR TITLE
feat: add css attribute selectors lab prototype

### LINKED ISSUE #400: chore: add lab project for attribute selectors to frontend cert
The Attribute selectors module will a lab for campers to do.
Labs are defined as short projects campers build on their own. It will consist of a blank editor and user stories.
Here is an example for the format:
https://www.freecodecamp.org/learn/the-odin-project/top-build-a-recipe-project/top-build-a-recipe-project

The previous workshop will have been the balance sheet project here
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-66

So you will need to create a small lab project that includes working with tabels and different attribute selectors. 

## note about the HTML boilerplate

At this point in the cert, campers will have created a few labs and worked on a few workshops.
So we no longer need to force them to write out the HTML boilerplate each time.
So we don't need to create user stories for the HTML boilerplate anymore. 

## Note about adding new images

For adding new images, make sure they are free to use. You will also need to be add that image to our CDN repo here
https://github.com/freeCodeCamp/cdn

Once the new image has been added to the CDN repo, then you will need to reference that cdn link in your project.

If you are not sure about if an image can be used, reach out on staff chat. 

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/css-projects` directory
- [ ] add your completed project in an `index.html` file and `styles.css` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #400 

In this case I would provide an example page to show how the final result should look like.

image

### CODE REVIEW

**Comment by zairahira** [comment]
As the lab is focused on CSS selectors, I think, the prototype should incorporate a few attribute selectors.

According to the project [sheet](https://docs.google.com/spreadsheets/d/1Kn3Ugrro5pingQhUWGLJhGc4olN4O3E8z72sMd615iY/edit?gid=0#gid=0), the lectures before this lab focus on:

- What is the attribute selector and how can it be used to target links with the `href` and `title` attribute? 
- How to use the attribute selector to target elements with the `lang` and `data-lang` attribute.
- How to use the attribute selector to target ordered list elements with the `type` attribute.

**Comment by jdwilkin4** [comment]
Yeah, I agree with Zaira here that we need to include at least one user story for an attribute selector since that is what this module is about.

So in the previous workshop here, they focused on the class attribute selector
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-66

So you can either add that or one of the other attribute selectors Zaira mentioned

**Comment by Dario-DC** [comment]
You're absolutely right, my bad. Sorry about the confusion.

I changed user stories to require class attribute selectors. Probably overcomplicated now, but at least it's pertinent to the topic.

**Comment by zairahira** [comment]
The prototype is looking great! Now there is plenty of practice available for attribute selectors.

**Review by zairahira — APPROVED** [review]

File: frontend-cert/css-projects/book-inventory/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+* {
+    box-sizing: border-box;
+    font-family: sans-serif;
```
Comment (zairahira) [inline frontend-cert/css-projects/book-inventory/styles.css:3]:
```suggestion
    font-family: Arial, sans-serif;
```

File: frontend-cert/css-projects/book-inventory/styles.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+td {
+    text-align: center;
+    padding: 0.3em;
+}
+tr[class="read"] {
+    background-image: linear-gradient(hsl(120, 100%, 85%, 0.60), hsl(120, 70%, 70%, 0.6));
+}
+tr[class="to-read"], tr[class="in-progress"] {
+    background-image: linear-gradient(hsl(120, 20%, 95%, 0.4), hsl(120, 20%, 75%, 0.4));
+}
```
Comment (zairahira) [inline frontend-cert/css-projects/book-inventory/styles.css:24]:
```suggestion
tr[class="to-read"] {
    background-image: linear-gradient(hsl(120, 20%, 95%, 0.4), hsl(120, 20%, 75%, 0.4));
}
tr[class="in-progress"] {
    background-image: linear-gradient(hsl(40, 100%, 85%, 0.60), hsl(40, 70%, 70%, 0.6));
}
```

**Review by zairahira — COMMENTED** [review]
Some optional changes:

1. The words are spaced out a bit in the current font-family `sans-serif`. You can replace the font with a readable one.
2. There should be a separate background color for in-progress books to differentiate the status.

**Review by jdwilkin4 — APPROVED** [review]

**Pull request merged**